core/platform/email/gae_email_services.py

```diff
@@ -22,6 +22,15 @@
 
 
 def get_incoming_email_address(reply_to_id):
+    """Gets the incoming email address. The client is responsible for recording
+       any audit logs.
```

Comment:
Remove these 4 spaces. We only indent continuation lines in case of 'Args' and 'Returns' section.

Reply:
Still needs addressing?